tools/ci/get_version.py

```
@@ -0,0 +1,86 @@
+    github_output = os.environ.get('GITHUB_OUTPUT')
+
+    version = ""
+    tag = ""
+    should_push_tag = False
+
+    if github_ref.startswith('refs/tags/'):
+        # 已由 tag 推送触发，直接使用该 tag 作为版本
+        version = github_ref[10:]
+        tag = version
+    elif not create_release:
+        # 非 release 构建，使用日期作为版本号 (UTC+8)
```

Ruff (0.14.8)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)